sonar-plugin-api/src/main/java/org/sonar/api/SonarRuntime.java

```diff
@@ -151,6 +151,12 @@
   Version getApiVersion();
 
   /**
+   * TODO
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1135)